calculate_costs.py

```diff
@@ -33,8 +33,11 @@ def read_simulation_csv(csv_file):
             # find values for parameter
             timestamp = datetime.datetime.fromisoformat(row["time"])
             price = float(row.get("price [EUR/kWh]", 0))
-            power_grid_supply = float(row["grid supply [kW]"])
-            power_fix_load = float(row["fixed load [kW]"])
+            power_grid_supply = float(row.get("grid supply [kW]", 0))
```

Comment:
If you set a default value for power_grid_supply I would recommend to also set a warning if no respective column is found, as it is required for all strategies.

Reply:
Is a warning necessary, @stefansc1 ? This column should always exists, right? I just addapted the `row.get(..., 0)` since it was used for all other columns.